test/GitHub.App.UnitTests/GitHub.App.UnitTests.csproj

```diff
@@ -0,0 +1,94 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I wanted to avoid treading on the work @meaghanlewis has been doing when we merge. Maybe we could merge the unit test refactoring first and then move this test into the new NUnit `UnitTests` project?